tools/flaky-test-reporter/config/config.yaml

```diff
@@ -75,6 +75,15 @@ jobConfigs:
     slackChannels:
       - name: eventing
         identity: C9JP909F0
+  - name: ci-knative-eventing-contrib-continuous
+    repo: eventing-contrib
+    type: postsubmit
+    issueRepo: eventing-contrib
+    slackChannels:
```

Comment:
It will be reported in these two channels **at the same time** right?

Reply:
yep, that's expected

Reply:
It seems you do not need this `ch` in https://github.com/knative/test-infra/blob/dfe8d3851bdb8f18e9d83e5ad9df1fbf1ca1d7ee/tools/flaky-test-reporter/slack_notification.go#L83. Guess it was written before we attended the Go concurrency class :)